lighthouse-viewer/app/src/main.js

```diff
@@ -43,3 +43,10 @@ Promise.all(loadPolyfillPromises).then(_ => {
   // eslint-disable-next-line no-new
   new LighthouseViewerReport();
 });
+
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.register('/sw.js', {scope: './'}).catch(e => {
```

Comment:
I disavow ever saying that 😄  This specific use case should be fine without providing a value for`scope`, so you might as well leave it out.

My recommendations, for the record: http://stackoverflow.com/a/33881341/385997